src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,91 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: string;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange = () => {},
+}) => {
+  const totalPages = Math.ceil(total / +perPage);
+  const numberOfPages = [];
+
+  for (let i = 1; i <= totalPages; i += 1) {
+    numberOfPages.push(i);
+  }
+
+  const isFirstPage = (): boolean => {
+    if (currentPage === 1) {
+      return true;
+    }
+
+    return false;
+  };
+
+  const isLastPage = (): boolean => {
+    if (currentPage === totalPages) {
+      return true;
+    }
+
+    return false;
+  };
+
+  return (
+    <ul className="pagination">
+      <li className={cn(
+        'page-item',
+        { disabled: isFirstPage() },
+      )}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={isFirstPage()}
+          onClick={() => !isFirstPage() && onPageChange(currentPage - 1)}
```

Comment:
no needs to check lat page at onClick since you disable this button

Reply:
originally I had code like this `onClick={() => onPageChange(currentPage - 1)}` it was enough as button disabled but due to tests failed I had to add condition.